react/tic-tac-toe/src/app/screens/Game/components/Match/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import styles from './styles.module.scss';
+
+const Match = (props) => {
+    const {player_one, player_two, winner} = props.match;
```

Comment:
As a convention we follow we use camelCase instead fo snake_case

Reply:
The problem is that this is howw they came from de API, should i change key names?